From 70c8239ee976b2c4325a77326f76bffd41502dbb Mon Sep 17 00:00:00 2001
From: Janmm14 <computerjanimaus@yahoo.de>
Date: Sun, 17 Jan 2016 17:22:05 +0100
Subject: [PATCH 4/7] Add origin and patcher git hash instead of patched git
 hash to version by using env variables which are set in build.sh

Remove scriptus maven plugin, as its no longer needed.
---
 pom.xml | 17 +----------------
 1 file changed, 1 insertion(+), 16 deletions(-)

diff --git a/pom.xml b/pom.xml
index 7ebde38..4bcf8fe 100644
--- a/pom.xml
+++ b/pom.xml
@@ -69,6 +69,7 @@
         <maven.compiler.source>1.7</maven.compiler.source>
         <maven.compiler.target>1.7</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <describe>git:${project.name}:${project.version}:origin-${origin_commit}-patcher-${patcher_commit}:${build.number}</describe>
     </properties>
 
     <dependencies>
@@ -101,22 +102,6 @@
     <build>
         <plugins>
             <plugin>
-                <groupId>net.md-5</groupId>
-                <artifactId>scriptus</artifactId>
-                <version>0.3.1</version>
-                <configuration>
-                    <format>git:${project.name}:${project.version}:%s:${build.number}</format>
-                </configuration>
-                <executions>
-                    <execution>
-                        <phase>initialize</phase>
-                        <goals>
-                            <goal>describe</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
-            <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>animal-sniffer-maven-plugin</artifactId>
                 <version>1.13</version>
-- 
1.9.5.msysgit.0

